src/Adapter/Category/Repository/CategoryRepository.php

```diff
@@ -149,8 +196,19 @@ public function assertCategoryExists(CategoryId $categoryId): void
      */
     public function getBreadcrumb(CategoryId $categoryId, LanguageId $languageId, string $separator = ' > '): string
```

Comment:
I don't think this should be injected in this because it's a Repository It already allows specifying a custom optional separator in the method so this one is fine

However, the separator could be injected in `CategoryDisplayNameBuilder` so it can use it as a parameter when it uses the repository